.github/workflows/production_deploy.yml

```
@@ -0,0 +1,130 @@
+    strategy:
+      matrix:
+        java-version: [ "21" ]
+        kotlin-version: [ "2.1.0" ]
+        distribution: [ "corretto" ]
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up JDK ${{ matrix.java-version }}
+        uses: actions/setup-java@v3
```

actionlint (1.7.7)

the runner of "actions/setup-java@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)